src/main/java/com/example/tour/place/ImageDTO.java

```diff
@@ -0,0 +1,14 @@
+package com.example.tour.place;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+@Data
+public class ImageDTO {
+    @JsonProperty(value= "id")
+    private Long id;
+    @JsonProperty(value= "path")
+    private String path;
+    @JsonProperty(value= "place")
+    private PlaceDTO place;
```

Comment:
Should the image take a place? What about the image of an activity?

Reply:
it's just to return the place object that the image belongs to if the front-end needs it